lib/matplotlib/axes/_axes.py

```diff
@@ -4613,9 +4613,12 @@ def stackplot(self, x, *args, **kwargs):
                          label_namer=None)
     def streamplot(self, x, y, u, v, density=1, linewidth=None, color=None,
                    cmap=None, norm=None, arrowsize=1, arrowstyle='-|>',
-                   minlength=0.1, transform=None, zorder=2, start_points=None):
+                   minlength=0.1, transform=None, zorder=None,
+                   start_points=None):
         if not self._hold:
             self.cla()
+        if zorder is None:
```

Comment:
I think this should be moved to the streamplot module, to have the function streamplot from there coherent with this one.
The documentation should also be updated.